src/blas/backends/mklcpu/mklcpu_level3.cxx

```diff
@@ -172,6 +177,9 @@ void gemm(cl::sycl::queue &queue, transpose transa, transpose transb, int64_t m,
             ::free(f32_b);
         });
     });
+#else
+    throw oneapi::mkl::unimplemented("blas", "cl::sycl::half", "when using hipSYCL");
```

Comment:
Please align this error message with more general one "half is disabled", I don't think it should be hipSYCL specific.